styles.css

```
@@ -2343,10 +2347,89 @@ body.multi-select-mode .track-item:hover {
+    padding: 0;
+    width: 10px;
+    align-self: stretch;
+}
+
+.rating-star.active {
+    color: var(--highlight, #f59e0b);
+    opacity: 1;
+}
+
+.rating-star.active .star-icon {
+    fill: currentColor;
```

Stylelint (17.4.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)